### PR TITLE
add docs for testing fx integration on local, dev, stage Relay

### PR DESCRIPTION
**Pull request by groovecoder**
How to test:

- [ ] Do the docs make sense?
- [ ] Try it

### CODE REVIEW

File: docs/fx-integration.md [hunk 1]
```diff
@@ -39,6 +42,26 @@ Firefox uses [Relay's REST API][relay-rest-api]. In particular, it uses these en
 - `GET /api/v1/profiles/` to get the user's Relay profile data.
 - `GET|POST /api/v1/relayaddresses/` to get and make relay addresses for the user.
 
+## Testing with local, dev, or stage Relay
+
+As of 2023-07-13, Relay integration is only enabled for 100% of Nightly users, so to
+test reliably you'll need to use Nightly. By 2023-08-29, it should be available for all
+Firefox users signed into the browser with their FXA.
```
Comment (jwhitlock) [inline docs/fx-integration.md:49]: This seems better. Is there a "proper" abbreviation for Firefox Accounts? I've always used "FxA", but these changes are FXA.

**Review by jwhitlock — CHANGES_REQUESTED** [review]
Some changes please. I'm trusting that you set up profiles correctly, since I've seen it on your laptop.

File: docs/fx-integration.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 # Firefox Integration
 
 As of Firefox 111, Relay has been integrated into the Firefox browser credential
-manager. This document describes the technical implementation.
+manager. It will show up for Firefox users who are signed into the browser with the FXA.
```
[the remark this answers is not in the document]
Comment (groovecoder) [reply docs/fx-integration.md:4]: whoops, meant to say "their FXA"

File: docs/fx-integration.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+This document describes the technical implementation.
+
+As of 2023-07-13, Relay integration is only enabled for 100% of Nightly users.
```
[the remark this answers is not in the document]
Comment (groovecoder) [reply docs/fx-integration.md:7]: changed to "As of 2023-07-13, Relay integration is only enabled for Nightly users"

File: docs/fx-integration.md (hunk 1 above)
Comment (groovecoder) [reply docs/fx-integration.md:49]: Changed to "FxA" everywhere.

**Review by jwhitlock — APPROVED** [review]
I appreciate the screenshot! Thanks for the docs!

**Pull request merged**